infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -249,12 +249,19 @@ private void fillHole(CorfuPayloadMsg<FillHoleRequest> msg, ChannelHandlerContex
         }
     }
 
+    /**
+     * Perform a prefix trim.
+     * Here the token is not used as is to perform the trim as the epoch at which the checkpoint was completed
```

Comment:
typo:  as is to -> to perform the trim, as...

Reply:
Done.